docs/store.js

```diff
@@ -91,13 +91,28 @@ var firebaseCalendarStorage = function(params) {
         var calTitle = calendarObj.state.title;
         var calState = calendarObj.state;
         
-        // Store the user's allCalendarIds and their calendarState .. setting the
-        // active calendar has its own method.
+        // Store the calendar into allCalendarIds first so that the user
+        // has permission to write into calendars/calUniqueId
```

Comment:
Maybe you should be allowed to create a new calendar as long as the final state is that the calendar is also in your users list of cal ids?